spec/support/matchers/csv_column_matcher.rb

```diff
@@ -0,0 +1,56 @@
+# Examples:
+# expect(report).to have_column("Quantity")
+# expect(report).to have_column("Quantity").with_value("1")
+# expect(report).to have_column("Quantity").with_values("1", "2")
+# expect(report).to have_column("Quantity").with_values(["1", "2"])
+RSpec::Matchers.define :have_column do |column_header|
+  match do |report|
+    lines = report.to_csv.split("\n")
+    headers = lines.shift.split(",")
+    @column_index = headers.index(column_header)
+    break false unless @column_index
+    break true unless @expected_values
+    @actual_values = lines.map { |line| line.split(",")[@column_index].to_s }
```

Comment:
I'd use Ruby's built-in `CSV` handling for this. Naïvely using `#split` is somewhat fraught, because your cells can technically contain internal commas, so long as they're wrapped in quotes.

The implementation using `CSV` also reads a lot better to me:

```ruby
csv = CSV.parse(report.to_csv)
break false unless csv.headers.include?(column_header)
break true unless @expected_values

@actual_values = csv.map { |row| row[column_header] }
@actual_values == @expected_values
```